azurerm/internal/services/network/client/client.go

```diff
@@ -16,6 +16,7 @@ type Client struct {
 	ExpressRouteCircuitsClient           *network.ExpressRouteCircuitsClient
 	ExpressRouteGatewaysClient           *network.ExpressRouteGatewaysClient
 	ExpressRoutePeeringsClient           *network.ExpressRouteCircuitPeeringsClient
+	FirewallPoliciesClient               *network.FirewallPoliciesClient
```

Comment:
Can we make this singular instead of plural?
```suggestion
	FirewallPolicyClient                 *network.FirewallPoliciesClient
```

Reply:
Will do.

Reply:
As other names of clients are plural, so I wonder is there any particular reason to make this client as singular?

Reply:
grammatically a singular name is more correct when reading the code. Some resources have made it through with plural names, which is unfortunate.

Reply:
I agree, the plural only matters in the context of REST. Is there also a suggestion to Go SDK (incl. track2) to rename the client?